Formula/switch-lan-play.rb

```diff
@@ -1,8 +1,10 @@
 class SwitchLanPlay < Formula
   desc "Make you and your friends play games like in a LAN"
   homepage "https://github.com/spacemeowx2/switch-lan-play"
-  url "https://github.com/spacemeowx2/switch-lan-play/archive/v0.0.7.tar.gz"
-  sha256 "0b645fa8bbff18b4b2d1dfd8d975790c6597bc0520c0ab1368387d08f8c2c900"
+  url "https://github.com/spacemeowx2/switch-lan-play.git",
+    :tag      => "v0.1.0",
+    :revision => "eda2c4dcb1db7948fcc0ae5d129fc4b8f4369ca2"
+  head "https://github.com/spacemeowx2/switch-lan-play.git"
```

Comment:
Please remove this line, we don't accept `HEAD` section additions in formula.

Reply:
Removed